configs/amazon-cloudwatch-publisher-osx.json

```diff
@@ -4,13 +4,7 @@
       "prefix": "osx",
```

Comment:
If we rename the Cognito config file with a `-cognito` suffix, let's rename this one with a `-credentials` suffix.

Reply:
i agree. making the change.